docs/_blog/roadmap-2023.md

```diff
@@ -0,0 +1,90 @@
+
+#### [RPC Client refactor](https://github.com/bitcoindevkit/bdk/pull/1041)
+
+A new blockchain client module, `bitcoind_rpc`, that allows wallets of any size to quickly sync "block by block".
+
+#### [Esplora Client refactor](https://github.com/bitcoindevkit/bdk/pull/1040)
+
+Adjustment of the Esplora backend module due to the new way of storing blocks (see above), with a related new cli example showing off new chain syncing features.
+
+#### [Removal of TransactionDetails from Wallet API](https://github.com/bitcoindevkit/bdk/pull/1048)
+
+Change on the retrieval of details of a transaction such as amounts, fee and fee rate in order to make it directly possible from the Wallet API. 
+
+### 1.0 alpha.3 milestone
+
+This third milestone focuses on wrapping up Wallet API changes in preparation for the 1.0.0 beta release. It is composed by many small changes regarding the following main topics:
+
+#### Better wallet errors
+
+The aim of this is to improve the readibility of different type of errors for the BDK users in order to be able to differentiate better in their own application.
+
+#### Further improving synching mechanism
+
+This includes creating a higher level method for syncing a Wallet using the new mechanisms recently added, improved the conflict resolution of unconfirmed transactions, some small bug fixes on the syncing mechanism and adding further examples for the different methodologies.
+
+### 1.0 beta release
+
+With the alpha.3 milestone underway, the plan is to release a BDK 1.0 beta release before the end of 2023. This BDK 1.0 beta version will provide a stable Wallet API our users can start experimenting with in their own application. During the beta phase the team will continue to add new code examples, documentation and testing. The team will also be updating the BDK-FFI mobile bindings (Kotlin/Android, Swift/iOS, python) to the new BDK 1.0 Wallet APIs (if you are more curious about BDK's general approach to bindings, this [this](https://bitcoindevkit.org/blog/bindings-scope/) blog post). New beta releases will be made as needed to address user feedback.
+
+### 1.0 release
+
+Once all beta user feedback, bug fixes, documentation, testing, and mobile language bindings are complete, the final BDK 1.0 will be ready to release. The purpose of calling this our 1.0 version is to signal to users that the main user facing APIs are stable and will be supported with bug fixes, documentation, and feature enhancements going forward. All projects using pre-1.0 versions of BDK should migrate to this version.
+
+### 1.1 release
+
+Current plans for a 1.0 release are still being developed but include plans to add compact block filter (BIP157/158) support and new electrum client features. No user facing Wallet APIs will be changed for this release, only new features will be added.
```

Comment:
I forgot that we also have some txbuilder fixes mentioned in bitcoindevkit/bdk#988 that wouldn't be breaking changes and could go in this release.

```suggestion
Current plans for a 1.0 release are still being developed but include plans to add compact block filter (BIP157/158) support and new electrum client features. We would also like to add some TxBuilder and coin selection enhancements. No user facing Wallet APIs will be changed for this release, only new features will be added.
```